modules/tasks/tasks/src/main/resources/webapp/js/directives.js

```diff
@@ -681,7 +681,7 @@
                 var manipulationType = $scope.type.toLowerCase();
                 $scope.manipulationTypes = [];
                 if(['unicode', 'string'].indexOf(manipulationType) > -1) {
-                    $scope.manipulationTypes = ['toUpper', 'toLower', 'capitalize', 'URLEncode', 'join', 'split', 'substring', 'format', 'parseDate'];
+                    $scope.manipulationTypes = ['toUpper', 'toLower', 'capitalizeFirstLetter', 'URLEncode', 'join', 'split', 'substring', 'format', 'parseDate'];
```

Comment:
I checked it and it works the same way before and after my changes but to prevent any mistakes i just changed name of message in messages.properties.